test_apps/stress_test/README.md

```diff
@@ -0,0 +1,31 @@
+# Datadog SDK Stress Test
+
+This app is meant to be used to stress test the Datadog Flutter SDK.
+
+It divides these tests into three areas:
+* Large payloads - adding large amounts of data to any Log `attributes` or `context` variables
+* High frequency - sending a lot of logs or RUM events
+* Mapping - adding mapping functions
+
+You can enable / disable the mapping functions in `main.dart`, and any of the tests.
```

Comment:
This might be confusing me because I'm not well versed in this app, but by this sentence are you saying you can enable/disable mapping functions in tests and `main.dart` or that you can enable/disable mapping functions and tests in the `main.dart`?

```suggestion
You can enable / disable the mapping functions and tests in `main.dart`.
```

Reply:
This is already changed locally. I actually just removed the 'and tests'.